src/annoylib.h

```diff
@@ -586,7 +733,8 @@ template<typename S, typename T, typename Distance, typename Random>
     n->children[1] = 0;
     n->n_descendants = 1;
 
-    for (int z = 0; z < _f; z++)
+    memset(n->v, 0, _f * sizeof(T));
```

Comment:
don't think this memset is needed
i guess the for loop could just be replaced by a memcpy though